src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -1033,6 +1034,42 @@ public static SendCommandEvent createSendCommandEvent(Cause cause, String argume
     }
 
     /**
+     * Creates a new instance of
+     * {@link org.spongepowered.api.event.command.CommandProcessEvent.Before}.
+     *
+     * @param cause The cause
+     * @param arguments The arguments
+     * @param command The command
+     * @return A new send command event
+     */
+    public static CommandProcessEvent.Before creatCommandProcessEventBefore(Cause cause, String arguments, String command) {
```

Comment:
If these were not automatically generated by the tool, they probably should be to match the other events.